### ISSUE TITLE
msal-angular 1.0, Google IdP - ClientAuthError: Token renewal operation failed due to timeout when user is signed in with multiple accounts

### ISSUE THREAD

**Issue by alvipeo** [AzureAD/microsoft-authentication-library-for-js#1967]
"msal": "^1.3.2",
"@angular/common": "~10.0.3",

I set up Google as an external IdP and when user is logged in with multiple Google accounts in a browser then I get this error when trying to issue a request to my web API:
> ClientAuthError: Token renewal operation failed due to timeout.

Here's the demo basically - [https://themoscow.video/demo/en/concerts](https://themoscow.video/demo/en/concerts). First, try to sign in with some Google services using multiple accounts (so the next time you sign in Google will present an account selection screen). Then go to this page, register and when you're logged in you should see the error in the console.

The code for the interceptor:

```
@Injectable()
export class AuthInterceptor implements HttpInterceptor {
  constructor(private msalSvc: MsalAuthService) {}

  intercept(
    req: HttpRequest<any>,
    next: HttpHandler
  ): Observable<HttpEvent<any>> {
    if (this.msalSvc) {
      return this.msalSvc.getGuestAccessToken().pipe(
        map((token) => {
          if (token) {
            const tokenValue = "Bearer " + token.accessToken;
            return req.clone({ setHeaders: { Authorization: tokenValue } });
          }
          return req;
        }),
        mergeMap((request) => next.handle(request)),
        catchError((error) => {
          console.error("error in the interceptor");
          console.dir(error);

          return throwError(error); // "Error occurred while preparing request to the server"
        })
      );
    } else {
      throw Error("Auth Service is undefined: NO auth header!");
    }
  }
}
```

**Comment by jo-arroyo** [comment]
@alvipeo Unfortunately, this a known issue that we have flagged with the B2C team. We will be tracking its progress in issue #1934, and will close this issue.

**Comment by DibyodyutiMondal** [comment]
I faced the same issue. I solved this (for v1.3) by passing in login_hint to to Google idp, like so:
![image](https://user-images.githubusercontent.com/50345677/88622518-9debdc80-d0c0-11ea-9353-db326d9c1dde.png)

Then in my outputClaims in the same TechnicalProfile, I used a separate claim `preferred_username` with PartnerClaimType as `email` and then passed that claim out in the token from the relying party policy as well.

Then I passed that claim in to the `loginHint` parameter like so in msal.js
```
this.msalInstance.acquireTokenSilent({
    scopes: appConfig.msal.scopes,
    loginHint: preferred_username
})
```

**Comment by alvipeo** [comment]
Hi! 

I opened a ticket for Azure AD B2C team, they're working on it.
